sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -117,7 +117,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
     </module>
 
     <!-- Check that paragraph tags are used correctly in Javadoc. -->
-    <module name="JavadocParagraph"/>
```

Comment:
I recognize some of the checks in this file as covered by ErrorProne as well. I've filed [[BEAM-4668]](https://issues.apache.org/jira/browse/BEAM-4668) to clean-up the redundant rules from ErrorProne.